packages/components/src/styles/_clear-fix.scss

```
@@ -3,7 +3,7 @@
 // Clear fix mixin (clearfix)
 // ==========================================================================
 
-@mixin clear-fix {
+@mixin a8c-clearFix {
```

According to [CSS/Sass guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#general-syntax-and-writing-rules), camelCase should be avoided in variables.

> Use hyphens, not underscores or camelCase, when naming things like IDs, classes, variable names, mixins, placeholders. Good: `.site-title`, Bad: `.siteTitle` or `.site_title`.

```suggestion
@mixin a8c-clear-fix {
```

---

I also noticed when [reviewing the guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#mixins-vs-extends), it seems the clearfix should be an `@extend`. I suspect that no longer holds true with our tree-shaken modular stylesheets 🙅‍♂ 

> * DO use `@extend` when in doubt, using the `%placeholder` syntax. This will produce leaner output.
> * DON'T use mixins for anything that doesn't accept an argument. This is `@extend` territory.
> * DO read [this article](http://miguelcamba.com/blog/2013/07/11/sass-placeholders-versus-mixins-and-extends/) if you don't understand `@extend`.

@jsnajdr Do you think this documentation section is worth revisiting?

---

With modular stylesheets, you should never extend from a "real" class, like `@extend .placeholder`. The extended class is unlikely to be present in the currently compiled CSS module.

`@extend %placeholder` is fine -- it's a usage of a reusable building block that doesn't output any CSS by itself. And the `%placeholder` definition will be in one of the "header" files that the compilation includes at the beginning.

The "leaner output" argument can be illustrated by this example:
```
%placeholder {
  color: grey;
}
.stats-placeholder {
  @extend %placeholder;
  width: 100%;
}
.reader-placeholder {
  @extend %placeholder;
  height: 100%;
}
```
This will output CSS that contains the `%placeholder` properties only once:
```
.stats-placeholder,  .reader-placeholder {
  color: grey;
}
.stats-placeholder {
  width: 100%;
}
.reader-placeholder {
  height: 100%;
}
```
Using a mixin would simply duplicate the properties:
```
.stats-placeholder {
  color: grey;
  width: 100%;
}
.reader-placeholder {
  color: grey;
  height: 100%;
}
```
This was an interesting optimization when the whole Calypso stylesheet was a single compilation unit. Then the styles were optimized globally. Now the deduplication is local to the currently compiled module, and no longer so effective.

> Do you think this documentation section is worth revisiting?

The part you quote still seems 100% true. It's just that the recommendation to prefer extend over mixins is much less important now.